### ISSUE TITLE
Remove deprecated ttl flag 

### ISSUE THREAD

**Issue by azdagron** [spiffe/spire#5254]
The CLI has long since deprecated the `-ttl` flag in favor of the SVID specific flags `-x509SVIDTTL` and `-jwtSVIDTTL`. The deprecated flag should be removed.

**Comment by ayushrakesh** [comment]
@azdagron I want to work on this issue, can you explain it more?

**Comment by amoore877** [comment]
@ayushrakesh see https://github.com/spiffe/spire/blob/main/cmd/spire-server/cli/entry/create.go#L51 , https://github.com/spiffe/spire/blob/main/cmd/spire-server/cli/entry/update.go#L48 

```
	// TTL for x509 and JWT SVIDs issued to this workload, unless type specific TTLs are set.
	// This field is deprecated in favor of the x509SVIDTTL and jwtSVIDTTL fields and will be
	// removed in a future release.
```

we want to remove `ttl` flag entirely in favor of the credential primitive-specific x509 / jwt flags. Probably `spire-server.md` also needs updating on this removal.

**Comment by heymarcel** [comment]
I'm working on this issue.

**Comment by heymarcel** [comment]
I've opened a PR: https://github.com/spiffe/spire/pull/5483

**Comment by azdagron** [comment]
Resolved by #5483.
